app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= provide :title, "Help" %>
+
+<h1>Help</h1>
+<p>
+Get help on the Ruby on Rails Tutorial at the 
```

Comment:
please learn about i18n
and use i18n for all text in view
https://github.com/ruby-i18n/i18n